src/presets/erc721.cairo

```diff
@@ -3,39 +3,64 @@
 
 /// # ERC721 Preset
 ///
-/// The ERC721 contract offers a batch-mint mechanism that
+/// The upgradeable ERC721 contract offers a batch-mint mechanism that
 /// can only be executed once upon contract construction.
+///
+/// For more complex or custom contracts, use Wizard for Cairo
+/// https://wizard.openzeppelin.com/cairo
 #[starknet::contract]
-mod ERC721 {
+mod ERC721Upgradeable {
+    use openzeppelin::access::ownable::OwnableComponent;
     use openzeppelin::introspection::src5::SRC5Component;
     use openzeppelin::token::erc721::ERC721Component;
-    use starknet::ContractAddress;
+    use openzeppelin::upgrades::UpgradeableComponent;
+    use openzeppelin::upgrades::interface::IUpgradeable;
+    use starknet::{ContractAddress, ClassHash};
 
+    component!(path: OwnableComponent, storage: ownable, event: OwnableEvent);
     component!(path: ERC721Component, storage: erc721, event: ERC721Event);
```

Comment:
```suggestion
     component!(path: ERC721Component, storage: erc721, event: ERC721Event);
    component!(path: OwnableComponent, storage: ownable, event: OwnableEvent);
```

Didn't we agreed on using the main module component first? If that's the case we should update the other presets as well.